### ISSUE TITLE
FIrefox. Dialogue history is not reflected

### ISSUE THREAD

**Issue by vladIsTrue** [Ajaxy/telegram-tt#341]
Firefix 121.0.1 
Windows 10

When switching from one chat to another, the dialogue history disappears. The browser window is in full screen mode

https://github.com/Ajaxy/telegram-tt/assets/115498650/eb4203c7-74d5-4524-a5a5-4ca9044237f5

**Comment by zubiden** [comment]
Do you have any extensions installed?

**Comment by vladIsTrue** [comment]
No. Only two plugins. Content decryption plugin Widevine (version
4.10.2710.0) and OpenH264 Video Codec provided by Cisco Systems (version
2.3.2)
